src/main/java/io/jenkins/plugins/orka/OrkaAgent.java

```diff
@@ -46,12 +53,15 @@ public class OrkaAgent extends AbstractCloudSlave {
     private String namePrefix;
     private String jvmOptions;
 
+    private final List<PortMapping> portMappings;
+
     @DataBoundConstructor
     public OrkaAgent(String name, String orkaCredentialsId, String orkaEndpoint, String vmCredentialsId,
             String node, String namespace, String namePrefix, String redirectHost, String image,
             Integer cpu, boolean useNetBoost, boolean useLegacyIO, boolean useGpuPassthrough, 
             int numExecutors, String host, int port, String remoteFS, boolean useJenkinsProxySettings, 
-            boolean ignoreSSLErrors, String jvmOptions, String memory, String tag, Boolean tagRequired)
+            boolean ignoreSSLErrors, String jvmOptions, String memory, String tag, Boolean tagRequired, 
+            List<PortMapping> portMappings)
```

Comment:
Check my comment for changing the singature of public methods: https://github.com/jenkinsci/macstadium-orka-plugin/pull/137#issuecomment-2478781360